app/i18n/en/conf.php

```diff
@@ -105,6 +105,16 @@
 		'none' => 'None',
 		'small' => 'Small',
 	),
+	'notification_timeout' => array(
+		'bad' => array(
+			'label' => 'Show warning banner',	// TODO
+			'seconds' => 'seconds (minimum 1)',	// TODO
+		),
+		'good' => array(
+			'label' => 'Show acknowledgement banner',	// TODO
+			'seconds' => 'seconds (0 means disabled notifications)',	// TODO
```

Comment:
Since there are no notifications in the label:
```suggestion
			'seconds' => 'seconds (0 means not shown)',
```

```suggestion
			'seconds' => 'seconds (0 means disabled)',
```